### ISSUE TITLE
Performance clarification

### ISSUE THREAD

**Issue by oak-test** [mljs/tree-set#1]
> Nevertheless adding, removing or finding an element is guaranteed to be performed in log(n).

Besides there is no removing operation, as I see from code finding is in O(log(n)), but adding is O(n).
That's the reason of why people call it "Tree"

**Comment by targos** [comment]
cc @andcastillo

**Comment by andcastillo** [comment]
@oak-test you are right, I forgot to implement the remove; and in the worst case the splice operation is O(N). Perhaps to implement a linked list deserve the time.

**Comment by targos** [comment]
FWIW there are many linked list implementations on npm
